socorro/lib/requestslib.py

```diff
@@ -35,18 +35,27 @@ def send(self, *args, **kwargs):
 
 def session_with_retries(
     total_retries=5,
-    backoff_factor=0.2,
+    backoff_factor=0.1,
     status_forcelist=(429, 500),
-    default_timeout=5.0,
+    default_timeout=3.0,
```

Comment:
After reading the urllib3 docs, I decided a default timeout of 5 is too high and we should use 3. In the case where the service is unresponsive, this kicks in for every retry. So if you have a max_retries of 5 and the default connection timeout is 5, that's 25 seconds sitting around waiting for something that's probably not going to come up in the 30 seconds gunicorn gives us before it gives up and kills the process.

Changing it to 3 reduces that to 15 seconds.